dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
I thought we decided not to do that right now since the user model doesn't have separate first and last name (aside from the ops serialized attributes which is a bit of a hack). We don't want to try guessing how to split the existing full name field into separate first and last. I agree we should head toward a comprehensive single identity per teacher, ideally in the user model. But it's not there yet.

The other issue is that we have a requirement that workshop pre-enrollment should not require login. So we ask for name & email, and then later match that with a user account (by email). Requiring this section confirmation and always creating an `enrollment` here will drastically simplify this, but it still leaves open the possibility of mismatched names. I'm open to suggestions. 

Is this ever something a teacher would want? Maybe Mr. Whatever in the user account display name for students to see, and the actual full name in workshop attendance so they get credit in the district?